src/chrome/content/rules/grafica.app.xml

```diff
@@ -0,0 +1,8 @@
+<!--
+grafica.app/ 
+-->
```

Comment:
Could you please clarify this comment?

Reply:
@zoracon I added this to highlight that the rule set will apply to all `grafica.app` urls, I can remove it if it feels unnecessary.

Reply:
@jumde Yes, it's not really needed here.

Reply:
@pipboy96 @zoracon - Updated.